include/utils/utils.php

```diff
@@ -3904,4 +3904,20 @@ function validateClickHouseSecret($signedvalue, $signedkey, $input) {
 	$computedSignature = base64_encode(hash_hmac('sha256', $input, $signedvalue, true));
 	return ($receivedSignature === $computedSignature);
 }
+
+function getFieldForAdvancedSearch($field, $module) {
```

Comment:
we have this function in Custom View: https://github.com/tsolucio/corebos/blob/master/modules/CustomView/CustomView.php#L530

please use that one